File: todo-app.java
import java.util.ArrayList;
import java.util.List;

public class TodoApp {
    private static List<String> todos = new ArrayList<>();

    public static void addTodo(String task) {
        todos.add(task);
    }

    public static void viewTodos() {
        for (int i = 0; i < todos.size(); i++) {
            System.out.println(i + 1 + " " + todos.get(i));
        }
    }

    public static void main(String[] args) {
        addTodo("Buy milk");
        addTodo("Finish homework");
        viewTodos();
    }
}

File: todo-app.js
let todos = []

const addTodo = task => {
    todos.push(task)
}

const viewTodos = () => {
    todos.forEach((todo, i) => {
        console.log(i + 1, todo)
    })
}

addTodo("Buy milk")
addTodo("Finish homework")
viewTodos()
